apps/web/src/pages/ManagerDashboardPage/Components/PoolCandidateSearchRequestPreviewListItem.tsx

```diff
@@ -0,0 +1,157 @@
+import { IntlShape, useIntl } from "react-intl";
+import { useState } from "react";
+
+import {
+  FragmentType,
+  getFragment,
+  graphql,
+  PreviewListItemFragment,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { PreviewList } from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+
+import { formatClassificationString } from "~/utils/poolUtils";
+
+import ReviewTalentRequestDialog from "./ReviewTalentRequestDialog";
+import { deriveChipSettings } from "../utils";
+
+const PreviewListItemPoolCandidateSearchRequest_Fragment = graphql(
+  /* GraphQL */ `
+    fragment PreviewListItem on PoolCandidateSearchRequest {
+      id
+      jobTitle
+      status {
+        value
+      }
+      requestedDate
+      applicantFilter {
+        qualifiedClassifications {
+          group
+          level
+        }
+      }
+      initialResultCount
+    }
+  `,
+);
+
+function buildTitle(request: PreviewListItemFragment, intl: IntlShape): string {
+  const classifications = unpackMaybes(
+    request.applicantFilter?.qualifiedClassifications,
+  );
+  const firstPart =
+    classifications.length == 1
+      ? formatClassificationString(classifications[0])
+      : null;
+
+  const secondPart =
+    request.jobTitle ?? intl.formatMessage(commonMessages.notProvided);
+
+  const completedTitle =
+    typeof firstPart === "string"
+      ? `${firstPart}${intl.formatMessage(commonMessages.dividingColon)}${secondPart}`
+      : secondPart;
+
+  return completedTitle;
+}
+
+interface PoolCandidateSearchRequestPreviewListItemProps {
+  poolCandidateSearchRequestQuery: FragmentType<
+    typeof PreviewListItemPoolCandidateSearchRequest_Fragment
+  >;
+  showUnfinishedPieces: boolean;
+}
+
+const PoolCandidateSearchRequestPreviewListItem = ({
+  poolCandidateSearchRequestQuery,
+  showUnfinishedPieces,
+}: PoolCandidateSearchRequestPreviewListItemProps) => {
+  const intl = useIntl();
+  const [dialogOpen, setDialogOpen] = useState<boolean>(false);
+  const request = getFragment(
+    PreviewListItemPoolCandidateSearchRequest_Fragment,
+    poolCandidateSearchRequestQuery,
+  );
+
+  const title = buildTitle(request, intl);
+
+  type MetaDataProps = React.ComponentProps<
+    typeof PreviewList.Item
+  >["metaData"];
+  type MetaDataPropItem = MetaDataProps[number];
+  const metaDataProps: MetaDataPropItem[] = [];
+  if (request.status?.value) {
+    const chipSettings = deriveChipSettings(request.status?.value, intl);
```

Comment:
You can remove the null check "?" here.

```suggestion
    const chipSettings = deriveChipSettings(request.status.value, intl);
```

Reply:
good catch!
[remove unnecessary nullsafe dereference](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11653/commits/cbe68f2195c5807fb2f032fa5c6dd3664e21fb59)